index.html

```diff
@@ -180,6 +180,20 @@ <h3>Publication Conformance</h3>
 
 			</section>
 
+			<section id="audio-type">
+				<h3>Publication Type</h3>
+
+				<p>The <dfn>Publication Type</dfn> is defined using the <code>type</code> term&#160;[[!pub-manifest]].</p>
+
+				<pre class="example" title="Setting a publication's type to Audiobook.">{
+    "@context" : ["https://schema.org", "https://www.w3.org/ns/wp-context"],
+    "type"     : "Audiobook"
+    …
+}</pre>
+
+				<p>If a <code>type</code> is not specified <a href="https://schema.org/Audiobook"><code>Audiobook</code></a>&#160;[[!schema.org]] is assumed as the default.</p>
+			</section>
```

Comment:
Should we go one (minor) step further and say that if the type _is_ specified but does not contain `Audiobook` then this is added to the types? If so, the corresponding check in the data validation (2) should also be slightly expanded.

Reply:
The only thing that gives me pause is that subtypes of Audiobook could always be defined at any time in schema.org. That would make a rule to add the parent unnecessary.

Reply:
That is true. The question is whether this is harmful or whether, potentially, have instances without any reference to audiobooks is harmful, and which one is more harmful, if at all.

Adding an Audiobook when there is a subtype is not harmful in the sense that it is simply a piece of unnecessary information. User agents may not use this at all (they would probably rely on conformsTo), search engines, however, may rely on typing.

I do not feel very strongly either way, to be perfectly honest. I am (weakly) leaning towards adding the type, but I could be convinced of the opposite...

@wareid ?

Reply:
I think adding the type, as you say, is at worst unnecessary information the user agent doesn't need, but could be useful for SEO. I'm ok with audiobooks always being added.

Reply:
I'm just concerned about adding things that the author intentionally didn't put in. It's one thing to have a default when a property is missing, but another to actively add information to what was authored.

I'm also a bit worried that this elevates `type` to a required property, as wouldn't the only way to enforce this be to have a statement that `type` must be set to Audiobook? We don't base any specific processing off it, so that seems problematic.

It may be too late for SEO, too. The property needs to be authored for a web crawler to index it.

Reply:
`conformsTo` is more important too, I see your point. I'm ok leaving this as is, I'm not attached to `type: "Audiobook"`

Reply:
Ok. Leave it as is.